addons/stock/models/stock_orderpoint.py

```diff
@@ -308,6 +308,11 @@ def _get_orderpoint_action(self):
         """
         action = self.env["ir.actions.actions"]._for_xml_id("stock.action_orderpoint_replenish")
         action['context'] = self.env.context
+        self._orderpoint_cleanup()
+        return action
+
+    @api.autovacuum
+    def _orderpoint_cleanup(self):
```

Comment:
I would only put the remove part in the autovaccum since the refill part has to be computed each time the report is open, it's useless to compute it one more time

Reply:
But unless until you click on the menu, new record will not get created. Even if you refresh the window!
The use case in mentioned on task.